theano/gof/cmodule.py

```diff
@@ -617,7 +617,17 @@ def __init__(self, dirname, check_for_broken_eq=True, do_refresh=True):
     Older modules will be deleted in ``clear_old``.
     """
 
-    def refresh(self, age_thresh_use=None, delete_if_problem=False):
+    def _get_module(self, name):
+        if name not in self.module_from_name:
+            _logger.debug('loading name %s', name)
+            self.module_from_name[name] = dlimport(name)
+            self.stats[1] += 1
+        else:
+            _logger.debug('returning compiled module from cache %s', name)
+            self.stats[0] += 1
```

Comment:
They weren't any more documented before so I'm not sure what they are.  I just tried to keep them at the same spots.